.github/workflows/release-branch.yml

```diff
@@ -110,6 +117,13 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.ARTIFACTS_STORAGE_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.ARTIFACTS_STORAGE_SECRET_ACCESS_KEY }}
+       - name: Init Application
+        run: devops/scripts/cf-app-init.sh
+      - name: Configure Env Vars (secrets)
+        run: |
+          cf set-env $APP_NAME REACT_APP_ECMPS_API_KEY $REACT_APP_ECMPS_API_KEY
+        env:
+          REACT_APP_ECMPS_API_KEY: ${{ secrets.ECMPS_API_KEY }}
```

Comment:
the api key needs to be set to the non production secret for all env other than production